### DISCUSSION TITLE
Parcel Doesn't Seem To Bundle/Minify Anything

### DISCUSSION THREAD

**Discussion by MykalMachon** [parcel-bundler/parcel#7608] [Q&A]
## Background
I'm trying to build a little web package that will eventually end up on NPM. Previously I've used a mish-mash of webpack, and typescript to achieve this. [I saw this page in parcel's docs](https://parceljs.org/getting-started/library/), and thought it would be a good choice for my package.

## What's the Issue?
It seems like parcel isn't bundling or minifying it's outputs. 
From what I can tell, by default parcel *should* bundle & minify all of my code into some distribution files on build. However, after running `parcel build` I'm getting some weird results. 

The files in my `./dist/` folder are essentially just JS versions of my typescript files with some weird variable names. 

***I wasn't sure if this was a bug, or just me missing something really simple, so I thought I would make a post here.***

### Example 
Here's a brief example of what my dist file looks like. 
```js
var $8zHUo$idbkeyval = require("idb-keyval");

function $parcel$export(e, n, v, s) {
  Object.defineProperty(e, n, {get: v, set: s, enumerable: true, configurable: true});
}

$parcel$export(module.exports, "init", () => $882b6d93070905b3$export$2cd8252107eb640b);

// * INTERNAL PRIVATE FUNCTIONS
/**
 * gets the form from cache and hydrates the page
 *
 * @param form the form to update in cache
 * @param store the custom store used by formstore
 * @returns Promise<Void>
 */ const $882b6d93070905b3$var$getFormFromCache = async (form, store)=>{
    const formId = form.id || 'unknown';
    const formKey = `form#${formId}`;
    // attempt to get the form from cache
    const formInCache = await $8zHUo$idbkeyval.get(formKey, store);
    // hydrate the HTML form
    formInCache.values.forEach((value)=>{
        const id = Object.keys(value)[0];
        const el = form.querySelector(`input#${id}, textarea#${id}`);
        el.value = value[id];
    });
};

// ... more code below
```
[See this repo for an example](https://github.com/MykalMachon/FormStore)  

## Resolution
Am I just missing something really simple here? or should I file an issue for this?
Thanks!

**Comment by joscha** [comment]
refs #4985
